### PR TITLE
Slideshow: Use gallery's images `size` if exists.

### PR DESCRIPTION
**Pull request by jeherve**
Replay of #2197

> Allow to display images in custom `size` as set in [gallery] tag - usable when using custom image size that was specially cropped and CSS is not enought to display in requested size.

cc @odie2

### CODE REVIEW

File: modules/shortcodes/slideshow.php [hunk 1]
```diff
@@ -125,6 +126,9 @@ function shortcode_callback( $attr, $content = null ) {
 		$attr['orderby'] = sanitize_sql_orderby( $attr['orderby'] );
 		if ( ! $attr['orderby'] )
 			$attr['orderby'] = 'menu_order ID';
+			
+		if ( ! $attr['size'] )
```
Comment (dereksmart) [inline modules/shortcodes/slideshow.php]: Let's add some braces here per WP code standards

File: modules/shortcodes/slideshow.php [hunk 2]
```diff
@@ -117,6 +117,7 @@ function shortcode_callback( $attr, $content = null ) {
 			'id'        => $post->ID,
 			'include'   => '',
 			'exclude'   => '',
+			'size'		=> '',
```
Comment (dereksmart) [inline modules/shortcodes/slideshow.php]: extra spaces

**Comment by dereksmart** [comment]
Fine to merge after minor adjustments.  Tested on both Jetpack and wpcom.

File: modules/shortcodes/slideshow.php (hunk 2 above)
Comment (jeherve) [inline modules/shortcodes/slideshow.php]: Done in 537c0793ec72f63adc2299e7b62f2392106766da

File: modules/shortcodes/slideshow.php (hunk 1 above)
Comment (jeherve) [inline modules/shortcodes/slideshow.php]: Done in 537c0793ec72f63adc2299e7b62f2392106766da

**Comment by jeherve** [comment]
Done!

**Comment by dereksmart** [comment]
Needs a rebase after merging #2201

**Comment by jeherve** [comment]
Done :+1:

**Comment by dereksmart** [comment]
:+1: :100: :+1:

**Pull request merged**